x11-proto/kbproto/kbproto-1.0.7.recipe

```diff
@@ -3,39 +3,40 @@ DESCRIPTION="Headers for the X11 KB protocol."
 HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
 COPYRIGHT="1993 by Silicon Graphics Computer Systems, Inc."
 LICENSE="MIT"
-REVISION="2"
-SOURCE_URI="http://xorg.freedesktop.org/releases/individual/proto/kbproto-1.0.7.tar.gz"
+REVISION="3"
+SOURCE_URI="https://xorg.freedesktop.org/releases/individual/proto/kbproto-$portVersion.tar.gz"
 CHECKSUM_SHA256="828cb275b91268b1a3ea950d5c0c5eb076c678fdf005d517411f89cc8c3bb416"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+ARCHITECTURES="any"
 
 PROVIDES="
 	kbproto$secondaryArchSuffix = $portVersion
-	devel:kbproto$secondaryArchSuffix = $portVersion
+	devel:kbproto = $portVersion
 	"
 REQUIRES="
-	haiku$secondaryArchSuffix
+	haiku
+	#lib:libxslt
 	"
 
 BUILD_REQUIRES="
-	haiku${secondaryArchSuffix}_devel
+	haiku_devel
+	#devel:libxslt
 	"
 BUILD_PREREQUIRES="
-	devel:util_macros$secondaryArchSuffix
-	cmd:autoconf
-	cmd:libtool
+	devel:util_macros
+	devel:xorg_sgml_doctools
 	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc
+	cmd:ld
 	cmd:make
-	cmd:gcc$secondaryArchSuffix
-	cmd:ld$secondaryArchSuffix
-	cmd:pkg_config$secondaryArchSuffix
+	cmd:pkg_config
+	#cmd:xmlto
 	"
 
 BUILD()
 {
-	libtoolize --force --copy --install
-	autoreconf -i
+	autoreconf -vfi
```

Comment:
I don't think this is necessary, given that we didn't modify the `configure` script